src/components/todo-item/index.js

```diff
@@ -2,15 +2,48 @@ import React from 'react';
 
 export default ({
   remove = e => console.log( 'no remove method defined' ),
+  toggle = e => console.log( 'no toggle method defined' ),
```

Comment:
you mean have a single action and an object as payload with different key:value pair to update an Item?